src/main/java/com/google/sps/servlets/DeleteMemberServlet.java

```diff
@@ -32,45 +32,52 @@
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    String userEmail = userService.getCurrentUser().getEmail();
 
+    // Query datastore with the current users email to see if they are in a family
```

Comment:
Instead of a comment, consider a helper function?

Comments in long functions that explain the different steps can indicate that you need a helper function.